lib/ome/common/xml/dom/Base.h

```diff
@@ -151,7 +151,29 @@ namespace ome
            */
           operator bool () const
           {
-            return get() != 0;
+            return get() != nullptr;
+          }
+
+          /**
+           * Check if the wrapped type is NULL.
+           *
+           * @returns true if valid, false if NULL.
```

Comment:
No, it's a copy paste error, and also confusing.  I've pushed a commit to correct and reword it.